pyproject.toml

```diff
@@ -37,15 +37,15 @@ prompt-toolkit = ">=3.0.0,<4.0.0"
 pyrsistent = ">=0.18.0,<1.0.0"
 typing-extensions = ">=4.7.0,<5.0.0"
 
-pytest = { version = ">=7.0.0,<9.0.0", optional = true }
+pytest = { version = ">=8.4.0,<9.0.0", optional = true }
```

Comment:
Do you think we could just check the [Pytest version tuple](https://docs.pytest.org/en/stable/reference/reference.html#pytest-version-tuple) in code before applying this rather than requiring 8.4.0+?

Reply:
Implemented option (2)